app/views/layouts/application.html.haml

```diff
@@ -6,12 +6,14 @@
     %meta{content: "width=device-width, initial-scale=1, shrink-to-fit=no", name: "viewport"}
     %meta{content: "Festive, Rugged, Unique Texas Trail Races", name: "description"}
     %meta{content: "Spectrum Trail Racing", name: "author"}
+    %csrf_meta_tags
```

Comment:
I had to add this line to get around a token error I was getting. Any reason this would be a bad idea or should we take another path to resolve? I'm currently reading up more about this.